build-system/tasks/prepend-global/index.js

```diff
@@ -108,7 +106,49 @@ function valueOrDefault(value, defaultValue) {
  * @return {!Promise<void>}
  */
 async function applyConfig(
-  config,
+  type,
+  target,
+  filename,
+  opt_localDev,
+  opt_localBranch,
+  opt_branch,
+  opt_fortesting,
+  opt_derandomize
+) {
+  const config = await getConfig(
+    target,
+    filename,
+    opt_localDev,
+    opt_localBranch,
+    opt_branch,
+    opt_fortesting,
+    opt_derandomize
+  );
+  const targetString = await fs.promises.readFile(target, 'utf8');
+  const fileString = config + targetString;
+  sanityCheck(fileString);
+  await writeTarget_(target, fileString, argv.dryrun);
+  const details =
+    '(' +
+    cyan(type) +
+    (opt_localDev ? ', ' + cyan('localDev') : '') +
+    (opt_fortesting ? ', ' + cyan('test') : '') +
+    (opt_derandomize ? ', ' + cyan('derandomized') : '') +
+    ')';
+  log('Applied AMP config', details, 'to', cyan(path.basename(target)));
```

Comment:
This message helps us ensure that we are applying the correct config to the correct files during `amp dist | build`. Now, the message isn't getting printed because we're using the banner / wrapper mechanism. I think it would be useful to move these two lines into `getConfig()`, and maybe reword the message _"Applied AMP config..."_ to _"Writing AMP config..."_. This way, both `applyConfig()` and `getAmpConfigForFile()` will print the same message under different use cases.

Reply:
Thank you for noticing this. I'll follow your advice here move it to `getConfig`. WDYT about the word _Creating AMP Config_ instead?

Reply:
Here's what the logging looks like today.

![image](https://user-images.githubusercontent.com/26553114/130830131-ca2b51ad-ad31-4d6b-9e0b-23d505169a4a.png)

_"Creating AMP config (prod, localDev, test) to v0.js"_ sounds weird.

If you don't like _"Writing AMP config..."_, how about making it _"Applying AMP config..."_?